packages/arb-token-bridge-ui/src/pages/api/withdrawals.ts

```diff
@@ -132,7 +132,7 @@ export default async function handler(
           l1Token,
           tokenAmount,
           isClassic,
-          l2BlockTimestamp,
+          timestamp: l2BlockTimestamp,
```

Comment:
I would keep this as it is in the subgraph, as it could cause confusion

Reply:
done